src/main/java/server/commands/CommandRooms.java

```diff
@@ -0,0 +1,30 @@
+package server.commands;
+
+import message.Message;
+import message.MessageType;
+import room.Room;
+import session.Session;
+
+import java.util.Set;
+
+public class CommandRooms implements Command {
+    private final Set<Room> rooms;
+
+    public CommandRooms(Set<Room> rooms) {
+        this.rooms = rooms;
+    }
+
+    private String getRooms() {
+        StringBuilder result = new StringBuilder();
+        for (Room room : rooms) {
+            result.append(room.toString()).append("\n");
+        }
```

Comment:
You can use Stream api and this collector.
https://docs.oracle.com/javase/8/docs/api/java/util/stream/Collectors.html#joining-java.lang.CharSequence-